python/nammu/utils/__init__.py

```diff
@@ -377,10 +380,9 @@ def copy_yaml_to_home(jar_file_path, source_rel_path, target_path):
             for zi in zf.infolist():
                 fn = zi.filename
                 if fn.lower() == source_rel_path:
-                    source_file = zf.open(fn)
-                    target_file = file(target_path, "wb")
-                    with source_file, target_file:
-                        shutil.copyfileobj(source_file, target_file)
+                    with zf.open(fn) as source_file:
+                        with file(target_path, "wb") as target_file:
```

Comment:
According to [the docs](https://docs.python.org/2/library/functions.html#file), it's better to use `open` instead of `file`. They do the same thing in Python 2, but `file` is intended more for type checking, and has been removed in version 3.